# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/MediaDecoder.h
# Commit: 85ef72ebde43
# Full Hash: 85ef72ebde439c5de258af43faa9bd83a6e23b26
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2019-08-29 21:45:25
# Regressor Bug: 1573102
# File Overlap Count: 1
# Description:
#   Bug 1573102 - Remove mGraph from HTMLMediaElement::OutputMediaStream and OutputStreamManager. r=karlt
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D43666
# ==============================================================================

diff -r f61dd73cac3a -r 85ef72ebde43 dom/media/MediaDecoder.h
--- a/dom/media/MediaDecoder.h	Thu Aug 29 13:32:31 2019 +0000
+++ b/dom/media/MediaDecoder.h	Thu Aug 29 13:32:50 2019 +0000
@@ -48,7 +48,7 @@
 class MediaFormatReader;
 class MediaDecoderStateMachine;
 struct MediaPlaybackEvent;
-class MediaStreamGraphImpl;
+class MediaStreamGraph;
 
 enum class Visibility : uint8_t;
 
@@ -176,7 +176,7 @@
   // Add an output stream. All decoder output will be sent to the stream.
   // The stream is initially blocked. The decoder is responsible for unblocking
   // it while it is playing back.
-  void AddOutputStream(DOMMediaStream* aStream, MediaStreamGraphImpl* aGraph);
+  void AddOutputStream(DOMMediaStream* aStream, MediaStreamGraph* aGraph);
   // Remove an output stream added with AddOutputStream.
   void RemoveOutputStream(DOMMediaStream* aStream);
 